src/aspire/commands/aspire.py

```diff
@@ -17,7 +16,6 @@
 from aspire.aspire.utils.viewstack import view_stack
 
 # TODO: Setting the random seed for numpy should be made a globally available option for all commands.
```

Comment:
Can you remove the comment as well?

Reply:
Done. I thought there should be a notice to remind us for including seed in commands.

Reply:
Thanks..no I think no command or library code should be setting the seed internally. If individual scripts/notebooks want to do so, they're free to do so. Let's leave #209 open for now though.